controller/rest/rest.go

```diff
@@ -1937,6 +1937,7 @@ func StartRESTServer(isNewCluster bool, isLead bool) {
 		// ReadTimeout:  time.Duration(5) * time.Second,
 		// WriteTimeout: time.Duration(35) * time.Second,
 		TLSNextProto: make(map[string]func(*http.Server, *tls.Conn, http.Handler), 0), // disable http/2
+		ErrorLog:     newHttpServerErrorWriter(),
```

Comment:
Should the server := &http.Server{} object in startFedRestServer() have the same fix?